mw-webapp/src/assets/icons/SearchIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import {IconProps} from "src/component/icon/Icon";
+
+/**
+ * Search icon
+ */
+export const SearchIcon = (props: IconProps) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 16 16"
+      fill="none"
```

Comment:
All our Icons has viewBox="0 0 24 24" . this property the same everywhere? How do you think? Or does it not affect anything?

Reply:
in figma maket our icon has 16px size. need ti reimport icon with 24x24 size. reimported

Reply:
We discussed this with designers today and decided  to use icons from feather/ Please, add icon for loop from feather too) Then all icons will be the same with the same properties)